### ISSUE TITLE
Duplicate description meta at runtime.

### ISSUE THREAD

**Issue by jordonbaade** [vuejs/vuepress#665]
#### Version

VuePress 0.12.0

#### Steps to reproduce

1. Install VuePress
2. Create a page and add extra meta data as per: 

https://vuepress.vuejs.org/guide/markdown.html#front-matter

```
---
meta:
  - name: description
    content: hello
  - name: keywords
    content: super duper SEO
---
```

3. Run `vuepress dev` or `vuepress build`
4. Check the page's meta tags

Was able to reproduce in fresh repo: 

https://github.com/jordonbaade/vuepress-0.12.0-meta-description-bug

#### What is expected?

A single:

`<meta name="description" content="description here">`

#### What is actually happening?

Two description meta elements:

![image](https://user-images.githubusercontent.com/840932/43093389-e555d404-8e7d-11e8-9919-81668274b989.png)

#### Other relevant information

If you check the build files directly the meta description element is only output once; seems the extra is injected on page view, and not just when using dynamic navigation.

- Your OS: macOS High Sierra v10.13.6
- Node.js version: v9.1.0
- Browser version: v67.0.3396.99
- Is this a global or local install? Global
- Which package manager did you use for the install? Yarn

**Comment by jordonbaade** [comment]
I just noticed some closed repos with the same/similar issue (#565 #112 ), but it wasn't clear to me if this was expected behavior after reading everything over. I will leave this up so someone more familiar can look it over and close it if it's expected.

**Comment by ulivz** [comment]
Sorry but I cannot reproduce it, here is my repo: https://github.com/ulivz/vuepress-665

See output files:

https://github.com/ulivz/vuepress-665/blob/ed5dd83f3f4d5d39e29b1cfa48b2f1a351259574/docs/.vuepress/dist/double-meta-description/index.html

https://github.com/ulivz/vuepress-665/blob/ed5dd83f3f4d5d39e29b1cfa48b2f1a351259574/docs/.vuepress/dist/single-meta-description/index.html#L5-L10

**Comment by jordonbaade** [comment]
For me the output files only contain a single element just like your examples there, so that's not a surprise, the extra is injected on page view (first page load, not just vue-router navigation).

![image](https://user-images.githubusercontent.com/840932/43242272-046f54d6-906e-11e8-918b-f44620ed87b1.png)

Was able to reproduce with your repo:

![image](https://user-images.githubusercontent.com/840932/43242487-5f02e31c-906f-11e8-9292-db38ee426b8d.png)

**Comment by ulivz** [comment]
Thanks for the clarification, there are truly two different issues.

#565 is for duplicate description meta at SSR, and what you said is in runtime.

Fixed it at b207a5f

**Comment by codeofsumit** [comment]
@ulivz while fixed for the `description` tag - the issue seems to persist for other meta tags:

```js
// config.js
  head: [
    ['meta', { name: 'twitter:card', content: 'summary_large_image' }],
    ['meta', { name: 'twitter:site', content: '@TweetsOfSumit' }],
    ['meta', { name: 'twitter:creator', content: '@TweetsOfSumit' }],
    ['meta', { property: 'og:type', content: 'article' }],
    ['meta', { property: 'og:title', content: 'Raum.sh' }],
    ['meta', { property: 'og:site_name', content: 'Raum.sh' }],
    ['meta', { property: 'og:url', content: 'https://raum.sh' }],
    [
      'meta',
      {
        property: 'og:image',
        content: 'https://raumsh-idpjzzfqhw.now.sh/images/speaking.jpg',
      },
    ],
  ],
```

``` md
# Markdown Front Matter

---
layout: Post
published: 2018-08-07
title: Blog is online 🙋‍♂️
meta:
  - property: og:title
    content: Blog is online 🙋‍♂️
  - name: description
    content: I finally decided to use VuePress as a blog engine.
  - name: keywords
    content: Web Developer Developer Relations Development Blog
---
```
(yup, social meta tags are a bit of a chaos but that's a different issue)

Output:
```html
  <title>Blog is online 🙋‍♂️ | Raum.sh</title>
  <meta name="description" content="I finally decided to use VuePress as a blog engine.">
  <meta name="twitter:card" content="summary_large_image">
  <meta name="twitter:site" content="@TweetsOfSumit">
  <meta name="twitter:creator" content="@TweetsOfSumit">
  <meta property="og:type" content="article">
  <meta property="og:title" content="Raum.sh">
  <meta property="og:site_name" content="Raum.sh">
  <meta property="og:url" content="https://raum.sh">
  <meta property="og:image" content="https://raumsh-idpjzzfqhw.now.sh/images/speaking.jpg">
  <meta name="title" content="Blog is online 🙋‍♂️">
  <meta property="og:title" content="Blog is online 🙋‍♂️">
  <meta name="keywords" content="Web Developer Developer Relations Development Blog">
```

Search for `og:title` - it's doubled.

Demo:
https://raumsh-vsumtazmko.now.sh/blog/BlogLaunch.html
